src/Pages/sunkyungnoh/Login/components/LoginForm.scss

```
@@ -0,0 +1,124 @@
+.loginForm {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin-top: 24px;
+
+  .enter {
+    position: relative;
+    margin-bottom: 8px;
+    width: 270px;
+    height: 38px;
+  }
+
+  input {
+    width: 270px;
+    height: 38px;
+    margin-bottom: 6px;
+    padding: 9px 0 7px 8px;
+    background-color: #fafafa;
+    border: 1px solid rgb(219, 219, 219);
+    border-radius: 4px;
+    outline: none;
+  }
+
+  .entered {
+    width: 266px;
+    height: 36px;
+    padding: 14px 0 2px 8px;
+  }
+
+  .form-description {
+    position: absolute;
+    top: 50%;
+    left: 8px;
+    transform: translateY(-50%);
+    color: #8e8e8e;
+    font-size: 12px;
+    text-align: center;
+    cursor: text;
+  }
+
+  .focused {
+    position: absolute;
+    top: 25%;
+    transition-property: font-size;
+    font-size: 7px;
+  }
+
+  .login.btn {
+    width: 268px;
+    height: 30px;
+    margin-top: 8px;
+    margin-bottom: 8px;
+    background-color: rgba(0, 149, 246, 0.3);
+    color: white;
+    font-size: 14px;
+    outline: none;
+    border-radius: 4px;
+    border-style: none;
+    cursor: pointer;
+  }
+
+  .active {
+    background-color: rgb(0, 149, 246) !important;
+  }
+
+  .loginOption {
+    width: 268px;
+  }
+
+  .loginOption > p {
+    display: flex;
+    align-items: center;
+    margin-top: 10px;
+    margin-bottom: 18px;
+    color: rgb(142, 142, 142);
+    font-size: 13px;
+  }
+
+  .loginOption > p::before {
+    content: '';
+    flex-grow: 1;
+    background: rgb(219, 219, 219);
+    height: 1px;
+    font-size: 0px;
+    line-height: 0px;
+    margin-right: 18px;
+  }
+
+  .loginOption > p::after {
+    content: '';
+    flex-grow: 1;
+    background: rgb(219, 219, 219);
+    height: 1px;
```

ㅎㅎㅎ 선경님도 역시 사람이군요! height를 먼저 적어주시면 더 좋을 것 같습니다! + margin-left도요!!
```suggestion
    height: 1px;
    background: rgb(219, 219, 219);
```

---

ㅋㅋㅋㅋ 저 완전 실수투성이인걸요..! ㅋㅋㅋ 감사합니당!